### PR TITLE
Bonk

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
When aiming for the head with a toolbox, there is a 1% chance of bonk.

## Why It's Good For The Game
Bonk

## Changelog
:cl:
add: bonk.ogg and a chance for the toolbox to use it
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
Did you test this? It looks like this wouldn't attack at all since it doesn't call `..()`.

File: code/game/objects/items/storage/toolbox.dm [hunk 1]
```diff
@@ -41,6 +41,13 @@
 	user.visible_message("<span class='suicide'>[user] robusts [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
+/obj/item/storage/toolbox/attack(mob/M, mob/user)
+	if(user.zone_selected == BODY_ZONE_HEAD)
+		if(prob(1))
+			playsound(loc, 'sound/items/bonk.ogg')
+		else
+			return
```
Comment (Jared-Fogle) [inline code/game/objects/items/storage/toolbox.dm:49]: Just don't put anything if you're not going to run any code after this.

**Comment by Paxilmaniac** [comment]
Pfft testing code, who even.

Actually let me check if it even does work.

**Comment by wesoda25** [comment]
About damn time

**Comment by Tumesuo** [comment]
Can you like link/send the sound cause checks don't have a sound diff bot.

**Comment by Iatots** [comment]
You can go to files changed and look at the raw sound file. 
It's a bamboo-ish hit sound, very cartoony.

**Comment by brotherbeyondd** [comment]
bonk

File: code/game/objects/items/storage/toolbox.dm [hunk 2]
```diff
@@ -41,6 +41,12 @@
 	user.visible_message("<span class='suicide'>[user] robusts [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
+/obj/item/storage/toolbox/attack(mob/M, mob/user)
+	..()
+	if(user.zone_selected == BODY_ZONE_HEAD)
+		if(prob(1))
+			playsound(loc, 'sound/items/bonk.ogg')
```
Comment (ATH1909) [inline code/game/objects/items/storage/toolbox.dm:48]: won't this play alongside the normal toolbox strike sound?

Comment (Paxilmaniac) [reply code/game/objects/items/storage/toolbox.dm:48]: Yes

**Review by nemvar — CHANGES_REQUESTED** [review]
You got a loicence for that sound?

**Comment by Paxilmaniac** [comment]
I don't actually know where the sound came from, so I probably don't have a loicence for that sound. Whatever the pr didn't really add anything interesting and I'm not gonna bother finding the source for it.

**Pull request closed without merging**